test/integration/transport.js

```diff
@@ -95,6 +117,42 @@ describe('given configurations for 10 nodes with address "127.0.0.1", WS ports 5
 					});
 				});
 
+				it('there should exactly 10 listening connections for 500[0-9] ports', done => {
+					utils.getListeningConnections(wsPorts, (err, numOfConnections) => {
+						if (err) {
+							return done(err);
+						}
+
+						if (numOfConnections === totalPeers) {
+							done();
+						} else {
+							done(
+								`There are ${numOfConnections} listening connections on web socket ports.`
+							);
+						}
+					});
+				});
+
+				it('there should maximum 180 established connections from 500[0-9] ports', done => {
```

Comment:
Please change the hardcoded numbers to rely on the introduced `totalPeers` constant.
like `[totalPeer * 9]`.